elixir/apps/api/lib/api/client/views/relay.ex

```diff
@@ -1,37 +1,32 @@
 defmodule API.Client.Views.Relay do
   alias Domain.Relays
 
-  def render_many(relays, expires_at) do
-    Enum.flat_map(relays, &render(&1, expires_at))
+  def render_many(relays, expires_at, conn_type) do
+    Enum.flat_map(relays, &render(&1, expires_at, conn_type))
   end
 
-  def render(%Relays.Relay{} = relay, expires_at) do
+  def render(%Relays.Relay{} = relay, expires_at, conn_type) do
     [
-      maybe_render(relay, expires_at, relay.ipv4),
-      maybe_render(relay, expires_at, relay.ipv6)
+      maybe_render(relay, expires_at, relay.ipv4, conn_type),
+      maybe_render(relay, expires_at, relay.ipv6, conn_type)
     ]
     |> List.flatten()
   end
 
-  defp maybe_render(%Relays.Relay{}, _expires_at, nil), do: []
+  defp maybe_render(%Relays.Relay{}, _expires_at, nil, _conn_type), do: []
 
-  defp maybe_render(%Relays.Relay{} = relay, expires_at, address) do
+  defp maybe_render(%Relays.Relay{} = relay, expires_at, address, conn_type) do
```

Comment:
STUN and TURN have different structures:
```suggestion
  defp maybe_render(%Relays.Relay{} = relay, expires_at, address, :turn) do
  defp maybe_render(%Relays.Relay{} = relay, expires_at, address, :stun) do
```